libs/generic-view/ui/src/lib/texts/headers.tsx

```diff
@@ -14,13 +14,24 @@ export const Header3: APIFC<undefined, HeaderConfig> = ({
   children,
   ...props
 }) => {
-  return <H3 {...props}>{isEmpty(children) ? config.text : children}</H3>
+  return (
+    <H3 $bold={config.bold} {...props}>
+      {isEmpty(children) ? config.text : children}
+    </H3>
+  )
 }
 
-export const H3 = styled.h3`
+export const H3 = styled.h3<{ $bold?: boolean }>`
```

Comment:
⚪ (suggestion)
Similar to the buttons, I suggest merging the H1, H2, H3, H4, and paragraph components into one integrated typography component. This way, adding an option like **bold** would automatically apply it to all elements. What do you think?

FYI @mkurczewski

Reply:
I already did some improvements in this topic. You can check the `headers.tsx` and `paragraphs.tsx`. However we can consider refactoring it in future. Feel free to add a task.